docker/start-notebook.sh

```diff
@@ -4,4 +4,15 @@
 
 set -e
 
+MYSQL_CMD="mysql -h $MYSQL_HOST -u root -p$MYSQL_ROOT_PASSWORD"
+
+until $MYSQL_CMD -e ";"; do
+  >&2 echo "MySQL is unavailable - sleeping"
+  sleep 1
+done
+
+$MYSQL_CMD -e "CREATE DATABASE IF NOT EXISTS $MYSQL_DATABASE_DATA;"
+$MYSQL_CMD -e "GRANT ALL PRIVILEGES ON *.* TO $MYSQL_USER;"
+$MYSQL_CMD -e "FLUSH PRIVILEGES;"
+
```

Comment:
Does luigi-import-worker need this as well? (and potentially luigid in the future once task history is set up)